import-export-cli/cmd/mi/activate/activate.go

```diff
@@ -0,0 +1,43 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package activate
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+)
+
+const activateCmdLiteral = "activate"
+const activateCmdShortDesc = "Activate artifacts deployed in a Micro Integrator instance"
+
+const activateCmdLongDesc = "Activate artifacts deployed in a Micro Integrator instance in the environment specified by the flag (--environment, -e)"
+
+const activateCmdExamples = utils.ProjectName + " " + utils.MiCmdLiteral + " " + activateCmdLiteral + " " + "endpoint" + " TestEP -e dev"
+
+// ActivateCmd represents the activate command
+var ActivateCmd = &cobra.Command{
+	Use:     activateCmdLiteral,
+	Short:   activateCmdShortDesc,
+	Long:    activateCmdLongDesc,
+	Example: activateCmdExamples,
+	Run: func(cmd *cobra.Command, args []string) {
+		utils.Logln(utils.LogPrefixInfo + activateCmdLiteral + " called")
+		cmd.Help()
+	},
+}
```

Comment:
Shall we add a new line to the end of the file? (It is a good practice to do [1]. Better to do the change where all applicable)
[1] https://www.quora.com/Why-is-it-a-common-practice-to-put-a-new-line-at-the-end-of-a-source-code-file

Reply:
I have already added the new line and it seems git is not showing it. If there is no new line git will show the :no_entry: at the end of the file AFAIK.

Reply:
No worries then. You can ignore this comment. Thanks!